### DISCUSSION TITLE
the scripting in cmd for weather changes

### DISCUSSION THREAD

**Discussion by alikazemian1998** [OpenModelica/OpenModelica#13249] [General]
Hello
as i want to to make changes for weather regarding guidance in command prompt, but it wrote: "'java' is not recognized as an internal or external command, operable program or batch file.". i appreciate if anyone could help me

![image](https://github.com/user-attachments/assets/102bc6da-3a7e-41f4-9232-a748f3a79f92)

**Comment by arun3688** [comment]
@alikazemian1998 I guess java path is not in your environment variable, Please make sure java is installed in your computer and add it to the `PATH` environment variable

**Comment by alikazemian1998** [comment]
Hi, thank you so much for your feedback. I installed Java, but when I want
to change the weather file, I face some problems, as attached. I would
appreciate if you could help me.

[image: image.png]
